### PR TITLE
Rewrite noe-core methods (Tomcat) to use TomcatConfigurator

### PR DESCRIPTION
**Pull request by PaulLodge**

### CODE REVIEW

File: core/src/main/groovy/noe/tomcat/configure/TomcatConfigurator.groovy [hunk 1]
```diff
@@ -39,6 +40,11 @@ class TomcatConfigurator {
 
   // - Methods ----------------------------------------------------------------------------------------
 
+  TomcatConfigurator keepDefaultAddressAndPort(boolean flag) {
```
Comment (jan-onderka) [inline core/src/main/groovy/noe/tomcat/configure/TomcatConfigurator.groovy:43]: I would rather see `set` keyword there instead of "overloading" variable.

**Comment by jan-onderka** [comment]
What is your motivation for implementing `TomcatConfigurator` as part of `Tomcat` class? It was intentionally put aside.

File: core/src/main/groovy/noe/tomcat/configure/TomcatConfigurator.groovy (hunk 1 above)
Comment (PaulLodge) [reply core/src/main/groovy/noe/tomcat/configure/TomcatConfigurator.groovy]: me too, nice catch - updated

**Comment by mmadzin** [comment]
I think the functions were not deprecated only because of the old way of modifying the configuration. 

Look at Tomcat 9 configuration for SSL connector: https://tomcat.apache.org/tomcat-9.0-doc/ssl-howto.html

**Pull request closed without merging**